src/converter/one/jfr/event/IEventAcceptor.java

```diff
@@ -0,0 +1,20 @@
+/*
+ * Copyright The async-profiler authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package one.jfr.event;
+
+public interface IEventAcceptor {
```

Comment:
I understand why you added this, but imo this is unnecessary level of abstraction.
Fortunately, as per my other comment, the only use of `IEventAcceptor` (and therefore `IEventAcceptor` itself) can be safely removed.